awx/api/views/__init__.py

```diff
@@ -4405,3 +4405,97 @@ def get_queryset(self):
         name = camelcase_to_underscore(attr)
         view = value.as_view()
         setattr(this_module, name, view)
+
+
+class WorfklowApprovalTemplateList(ListAPIView):
+
+    model = models.WorkflowApprovalTemplate
+    serializer_class = serializers.WorkflowApprovalTemplateSerializer
+
+    def get(self, request, *args, **kwargs):
+        if not request.user.is_superuser and not request.user.is_system_auditor:
+            raise PermissionDenied(_("Superuser privileges needed."))
+        return super(WorfklowApprovalTemplateList, self).get(request, *args, **kwargs)
+
+
+class WorfklowApprovalTemplateDetail(RetrieveAPIView):
+
+    model = models.WorkflowApprovalTemplate
+    serializer_class = serializers.WorkflowApprovalTemplateSerializer
+
+
+class WorkflowApprovalTemplateNotificationTemplatesAnyList(SubListCreateAttachDetachAPIView):
+
+    model = models.NotificationTemplate
+    serializer_class = serializers.NotificationTemplateSerializer
+    parent_model = models.WorkflowApprovalTemplate
+
+
+class WorkflowApprovalTemplateNotificationTemplatesStartedList(WorkflowApprovalTemplateNotificationTemplatesAnyList):
+
+    relationship = 'notification_templates_started'
+
+
+class WorkflowApprovalTemplateNotificationTemplatesErrorList(WorkflowApprovalTemplateNotificationTemplatesAnyList):
+
+    relationship = 'notification_templates_error'
+
+
+class WorkflowApprovalTemplateNotificationTemplatesSuccessList(WorkflowApprovalTemplateNotificationTemplatesAnyList):
+
+    relationship = 'notification_templates_success'
+
+
+class WorkflowApprovalTemplateLaunch(GenericAPIView):
```

Comment:
Is this a thing we actually want or need?  I can't think of a reason why you'd want to *actually* manually launch one of these.

Reply:
Oops, I forgot to take this out when editing/cleaning up the `awx/api/urls/workflow_approval_template.py` file!